packages/lesswrong/lib/collections/posts/schema.tsx

```diff
@@ -1453,7 +1475,7 @@ const schema: SchemaType<DbPost> = {
       foreignPostId: { type: String, optional: true, nullable: true },
     }),
     optional: true,
-    nullable: true,
+    nullable: true, //TODO not-null, why explicitly set?
```

Comment:
A field for the EA forum to chime in on

Reply:
I think this is only null for posts created before we implemented crossposting. I think either leaving it nullable or just backfilling it with `{"isCrosspost":false}` is fine - I don't have a strong preference.

Reply:
I've added a default value of false so it can be backfilled